packages/react-notion-x/src/block.tsx

```diff
@@ -531,20 +531,20 @@ export const Block: React.FC<BlockProps> = (props) => {
     case 'excalidraw':
     // fallthrough
     case 'image':
-    // fallthrough
+      return <AssetWrapper blockId={blockId} block={block} darkMode={darkMode} />
```

Comment:
No need to pass down `darkMode` as it's available from the react context.

Reply:
I removed the return of the AssetWrapper and fall through everything.
I also removed the "darkMode" and the new "vercelImages" and used the notion context.